package.json

```diff
@@ -86,7 +86,7 @@
     "gl-mat4": "^1.2.0",
     "gl-mesh3d": "^2.3.1",
     "gl-plot2d": "^1.4.5",
-    "gl-plot3d": "^2.4.6",
+    "gl-plot3d": "git://github.com/jdpaterson/gl-plot3d.git#8f96f0362366394a188b2f7451faf4b48da85433",
```

Comment:
Please bump `gl-plot3d` version to "^2.4.7" here. Save and `npm install`. 
Then commit `package.json` and `package-lock.json`.
Thanks!

Reply:
Done. Thanks for your help @archmoj !